app/build.gradle

```diff
@@ -65,4 +65,6 @@ dependencies {
 
     implementation Dependencies.lifecycle_extensions
     implementation Dependencies.lifecycle_viewmodel
+
+    implementation Dependencies.viewpager2
```

Comment:
The Dependencies are grouped - can you shift the viewpager2 dependency to line 50?